sdk/ai/azure-ai-projects/azure/ai/projects/aio/_patch.py

```diff
@@ -213,13 +214,17 @@ def __init__(  # pylint: disable=super-init-not-called,too-many-statements
         self.evaluations = EvaluationsOperations(self._client3, self._config3, self._serialize, self._deserialize)
         self.inference = InferenceOperations(self)
 
+    async def _close_credential_maybe(self):
```

Comment:
The credential can be shared amongst multiple clients. In other words, passing a credential to the client does not mean that the caller is handing off ownership.

Reply:
@johanste could we handle the credential closing issue in the separate PR as we also will need to change the async samples?

Reply:
Fixed.